pkg/odo/cli/utils/utils.go

```diff
@@ -15,19 +13,17 @@ const RecommendedCommandName = "utils"
 // NewCmdUtils implements the utils odo command
 func NewCmdUtils(name, fullName string) *cobra.Command {
 	terminalCmd := NewCmdTerminal(terminalCommandName, util.GetFullName(fullName, terminalCommandName))
-	configurationCmd := config.NewCmdConfiguration(config.RecommendedCommandName, util.GetFullName(fullName, config.RecommendedCommandName))
 	utilsCmd := &cobra.Command{
 		Use:   name,
 		Short: "Utilities for terminal commands and modifying Odo configurations",
 		Long:  `Utilities for terminal commands and modifying Odo configurations`,
```

Comment:
This needs to be changed. Also, maybe it would make sense to remove the `utils` command altogether if it only has one sub-command?

Reply:
@kadel would love your opinion here

Reply:
I would keep it for now. We can remove it later.